### PR TITLE
Sbo blog update draft

### PR DESCRIPTION
**Pull request by mezarin**
Thsi PR removes Service Binding Operator mentions. This was triggered by ODO's temporary removal (ODO 2.4.0) of the support for linking applications and services using the Service Binding Operator. ODO has its own application/service linking capability based on SBO APIs; therefore, for the time being, this is what this blog will showcase.

### CODE REVIEW

File: posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc [hunk 1]
```diff
@@ -12,10 +12,9 @@ open-graph-image: https://openliberty.io/img/blog/ol-stack-jpa-app-db-bind-brows
 = Bind an operator-managed PostgreSQL database to a JPA application in a Kubernetes cluster
 Edward Mezarina <https://github.com/mezarin>
 
-Binding your microservice application to operator-based services, such as a database, is now easier, thanks to the Service Binding Operator and OpenShift Do (odo). These frameworks enable the automatic collection of service data and easier resource data sharing, while abstracting resource configuration and deployment details.
-
-In this post, we’ll use the Open Liberty devfile stack, Openshift odo, and the Service Binding Operator to easily deploy an application that uses the Java Persistence API (JPA) and bind a PostgreSQL database to the application.
+Binding your microservice application to operator-based services, such as a database, is now easier, thanks to OpenShift Do (odo). This framework enables the automatic collection of service data and easier resource data sharing, while abstracting resource configuration and deployment details.
 
+In this post, we’ll use the Open Liberty devfile stack and Openshift odo to easily deploy an application that uses the Java Persistence API (JPA) and bind a PostgreSQL database to the application.
```
Comment (scottkurz) [inline posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc]: Openshift => OpenShift

File: posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc [hunk 2]
```diff
@@ -151,7 +116,7 @@ metadata:
     service.binding/db_user: 'path={.spec.databaseUser}'
 ----
 
-Adding the annotations ensures that the Service Binding Operator will inject the `databaseName`, `databasePassword` and `databaseUser` spec values into the application. Note that the instance name you configure will be used as part of the name of various artifacts and resource references. Be sure to change it.
+Adding the annotations ensures that odo will inject the `databaseName`, `databasePassword` and `databaseUser` spec values into the application. Note that the instance name you configure will be used as part of the name of various artifacts and resource references. Be sure to change it.
```
Comment (scottkurz) [inline posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc]: I think we need to at least make a nod to the fact that there's a mapping happening under the covers...before by SBO, now by odo itself, to get to  server.xml vars like:
`${DATABASE_DB_USER}`

**Review by scottkurz — CHANGES_REQUESTED** [review]

File: posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc [hunk 3]
```diff
@@ -116,7 +116,9 @@ metadata:
     service.binding/db_user: 'path={.spec.databaseUser}'
 ----
 
-Adding the annotations ensures that odo will inject the `databaseName`, `databasePassword` and `databaseUser` spec values into the application. Note that the instance name you configure will be used as part of the name of various artifacts and resource references. Be sure to change it.
+The instance name you configure will be used as part of the name of various artifacts and resource references. Be sure to change it.
```
Comment (scottkurz) [inline posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc]: OK.. I was confused.. I see what you're saying.

How about: 
> Be sure to change the metadata name from the generated value to 'sampledatabase' in order to match the instructions below.

**Review by scottkurz — DISMISSED** [review]
Suggested change.

File: posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc (hunk 3 above)
Comment (mezarin) [reply posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc]: done

File: posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc (hunk 1 above)
Comment (mezarin) [reply posts/2021-09-17-binding-app-to-postgresql-db-with-odo-and-ol-stack.adoc]: done

**Comment by mezarin** [comment]
Hi @dmuelle, it looks like the draft branch has not caught up with the prod branch. Could you please sync those two branches before this PR is merged. Thanks!.

**Comment by dmuelle** [comment]
> Hi @dmuelle, it looks like the draft branch has not caught up with the prod branch. Could you please sync those two branches before this PR is merged. Thanks!.

resolved the conflict, should be clear now

**Review by dmuelle — APPROVED** [review]
LGTM- the blog tag commits are ok now that the conflict is resovled

**Pull request merged**